app/presenters/summary/components/funding_decision.rb

```diff
@@ -0,0 +1,64 @@
+module Summary
+  module Components
+    class FundingDecision < BaseRecord
+      private
+
+      def answers # rubocop:disable Metrics/AbcSize,Metrics/MethodLength,Metrics/CyclomaticComplexity,Metrics/PerceivedComplexity
+        [
+          Components::FreeTextAnswer.new(
+            :funding_decision_maat_id, funding_decision.maat_id,
+            show: funding_decision.maat_id.present?
```

Comment:
You can remove the `show:` argument as this is the default behaviour:
https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/blob/main/app/presenters/summary/components/base_answer.rb#L25-L31